src/tic_tac_toe/cli_ui.clj

```diff
@@ -0,0 +1,118 @@
+(ns tic-tac-toe.cli-ui
+  (:require [tic-tac-toe.messages :refer [ask-for-tile-choice
+                                          ask-replay
+                                          draw-message
+                                          goodbye
+                                          invalid-mode
+                                          picked-tile-message
+                                          not-integer-message
+                                          not-y-or-n
+                                          welcome-message
+                                          winner-message]]))
+
+(defn clear-screen []
+  (print (str (char 27) "[2J")))
+
+(defn delay-in-secs 
+  [secs]
+  (Thread/sleep (* secs 1000)))
+
+(defn format-board-cli
+  [board-values]
+  (let [board-size (int (Math/sqrt (count board-values)))]
+    (->> board-values
+         (map (fn [x] (str x " ")))
+         (partition board-size)
+         (interpose "\n")
+         (apply concat)
+         clojure.string/join)))
+
+(defn send-message
```

Comment:
Is this function just supposed to return a print statement? If so, I find the naming is a little confusing. Something like `display-message` may be clearer if I understood what it does correctly.